content/en/smart-contracts/filecoin-evm-runtime/differences-with-ethereum/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Differences with Ethereum"
+title: "Differences to Ethereum"
```

Comment:
No, I'm pretty certain it's not.

Example sentences:
- The difference with my new sneakers is that they don't squeak
- The difference to my old sneakers is that my new sneakers don't squeak.

You use "With" when stating a characteristic of a thing that is unique or different from other similar objects. No real comparison is being made...and you use "To" or "From" when you _do_ make a comparison. In this case you _are_ trying to make a comparison between Ethereum and Filecoin.

The only other way I could see writing this is saying the "Difference with Filecoin" and then describe what makes using Filecoin different. 

Does that make sense?

Reply:
...I'll throw a third option in the mix, instead of "to" and "with," it should probably be "between" and mention both items being compared.  The attempt to abbreviate the title by not mentioning Filecoin has lead to confusion --title should read "Differences between Filecoin and Ethereum."

- The difference to my new sneakers is that my new sneakers don't squeak --> should be "The difference between my old sneakers and my new pair are that the latter don't squeak."

Reply:
I'm not gonna fight about grammar. If you really want to make this change @lksbrssr you need to:

- [ ] Update the file and folder names to reflect the new title.
- [ ] Add an alias redirect so users don't get lost.
- [ ] Update any internal links to this page (Hugo should flag any links that need changing).

Reply:
I like "Differences between Filecoin and Ethereum"; maybe tweak to "Differences between Filecoin and Ethereum's EVM". 

I'm not trying to fight about grammar either?

Reply:
Another shorter option: "Comparison to Ethereum" or "Compare to Ethereum".

Reply:
upvoting "Differences between Filecoin and Ethereum's EVM"